DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -417,37 +417,63 @@ public void
         }
 
         [Test]
-        public void RemoveDelegateFromCourse_removes_delegate_from_course()
+        public void DelegateHasCurrentProgress_returns_true_if_delegate_has_current_progress()
         {
             // Given
             A.CallTo(() => progressDataService.GetDelegateProgressForCourse(1, 1)).Returns(
                 new List<Progress> { new Progress { ProgressId = 1, Completed = null, RemovedDate = null } }

```

Comment:
Could we also make this return a completed progress record and a removed progress record just to make sure those don't interfere.